File: Android applications/p0261intentfilter/java/com/korobeynikov/p0261intentfilter/ActivityTime.java
package com.korobeynikov.p0261intentfilter;

import java.sql.Date;
import java.text.SimpleDateFormat;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

public class ActivityTime extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.time);
        SimpleDateFormat sdf=new SimpleDateFormat("HH:mm:ss");
        String time=sdf.format(new Date(System.currentTimeMillis()));
        TextView tvTime=findViewById(R.id.tvTime);
        tvTime.setText(time);
    }
}

File: KotlinLessons/AndroidLessons/p0261intentfilter/java/com/korobeynikov/p0261intentfilter/ActivityTime.kt
package com.korobeynikov.p0261intentfilter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.time.*
import java.text.SimpleDateFormat
import java.util.*

class ActivityTime : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.time)

        val sdf=SimpleDateFormat("HH:mm:ss")
        val time=sdf.format(Date(System.currentTimeMillis()))
        tvTime.text=time
    }
}